packages/client/hmi-client/src/components/articles/Document.vue

```diff
@@ -11,12 +11,13 @@
 						<a
 							:href="`https://doi.org/${doi}`"
 							rel="noreferrer noopener"
+							target="_blank"
```

Comment:
Let's remove this, unless it was specified by Jamie, Holly or Pascale.

Reply:
This one is part of the requirement :)